portal/messaging/messages.php

```diff
@@ -58,18 +58,15 @@
     $dashuser = getUserIDInfo($_SESSION['authUserID']);
 }
 
-function getAuthPortalUsers()
-{
+function getAuthPortalUsers() {
     $resultpd = $resultusers = $resultpatients = array();
     if (IS_DASHBOARD) { // admin can mail anyone
-        $authusers = sqlStatement("SELECT users.username as userid,
- CONCAT(users.fname,' ',users.lname) as username, 'user' as type FROM users WHERE authorized = 1");
+        $authusers = sqlStatement("SELECT users.username as userid, CONCAT(users.fname, ' ', users.lname) as username, 'user' as type FROM users WHERE facility_id != 0 AND active = 1");
```

Comment:
Clever but still iffy. I'm going to add a portal provider flag to Users while adding another drop down so, hold off till then(maybe today). Having a separate flag for those allowed to access Portal Dashboard just makes the best sense....

Reply:
@sjpadgett Yeah my intention with this change was to allow replies not to just doctors (which would have the auth flag checked) but to also nurses and other people using the EMR. It's weird that we don't have a flag in the system to distinguish them yet if I also do a fix for #3561 that wouldn't need to be created.